packages/highlight/src/lib/themes/poimandres/poimandres.ts

```diff
@@ -0,0 +1,84 @@
+import {HighlightStyle, syntaxHighlighting} from '@codemirror/language';
+import {tags as t} from '@lezer/highlight';
+import {defineEditorTheme} from '../../core';
+
+export const palette = {
+  brightYellow: '#fffac2',
+  brightMint: '#5DE4c7',
+  lowerMint: '#5fb3a1',
+  blueishGreen: '#42675A',
+  lowerBlue: '#89ddff',
+  lightBlue: '#ADD7FF',
+  desaturatedBlue: '#91B4D5',
+  bluishGrayBrighter: '#7390AA',
+  hotRed: '#d0679d',
+  pink: '#f087bd',
+  gray: '#a6accd',
+  lighterGray: '#c0cad5',
+  darkerGray: '#767c9d',
+  bluishGray: '#506477',
+  focus: '#303340',
+  bg: '#1b1e28',
+  offWhite: '#e4f0fb',
+  selection: '#717cb4',
+  white: '#ffffff',
+  black: '#000000',
+};
+
+export const poimandres = [
+  syntaxHighlighting(
+    HighlightStyle.define([
+      {tag: t.definitionKeyword, color: palette.desaturatedBlue},
+      {tag: t.controlKeyword, color: palette.lowerMint},
+      {
+        tag: t.attributeName,
```

Comment:
This can be replaced with the `attrName` property present in the `highlight` object of `defineEditorTheme` function which manages it already that tag